src/lib/page-action.js

```diff
@@ -10,6 +10,12 @@ export default class PageAction {
     browser.webNavigation.onCompleted.addListener(({ frameId, tabId }) => {
       this.create(frameId, tabId);
     });
+    browser.webNavigation.onHistoryStateUpdated.addListener((
+      { frameId, tabId }
+    ) =>
```

Comment:
Uh oh! I had hoped that prettier would do Exactly What I Want! Sigh... Why doesn't everyone in the world have the same taste I do?